docs/getting-started-guides/kubeadm.md

```diff
@@ -195,6 +195,48 @@ Once a pod network has been installed, you can confirm that it is working by che
 
 And once the `kube-dns` pod is up and running, you can continue by joining your nodes.
 
+If you see the following status 
```

Comment:
Now that I have reread this, I think these two pieces should be in one sentence:

You may have trouble in the configuration if you see the following statuses: